packages/muelu/adapters/amgx/MueLu_AMGXOperator_decl.hpp

```diff
@@ -362,7 +362,7 @@ namespace MueLu {
           recv_maps[i] = &(recvDatas[i][0]);
```

Comment:
This could be bad if for some reason, another process were listed as a receive neighbor but didn't have any incoming data.  `std::vector::data()` need not return `nullptr` if the vector is empty.